planetiler-custommap/src/main/resources/samples/shortbread.yml

```diff
@@ -80,6 +82,8 @@ layers:
     - *name
     - *name_en
     - *name_de
+    - key: way_area # area in square meter (Mercator Projection)
+      value: "${ feature.area('km2') * 1000000.0 }"
```

Comment:
switch to `"${ feature.area('m2') }"` too?

Reply:
```suggestion
      value: "${ feature.area('m2') }"
```